File: we_5_3_the_coriolis_correction_in_the_velocity.py
# Integrated Navigation Systems — Chapter 5: Inertial Navigation Mechanization
# Worked Example 5.3 — The Coriolis correction in the velocity update
# Companion code (educational reference implementation; see the book for context).

import numpy as np
wie=7.2921150e-5; L=np.radians(40); wn=np.array([wie*np.cos(L),0,-wie*np.sin(L)]); v=np.array([100.,0,0])
print(round(2*np.cross(wn,v)[1]*1000,4),'mm/s^2')  # -9.3746

File: we_5_3_the_coriolis_correction_in_the_velocity.m
% Integrated Navigation Systems — Chapter 5: Inertial Navigation Mechanization
% Worked Example 5.3 — The Coriolis correction in the velocity update
% Companion code (educational reference implementation; see the book for context).

wie=7.2921150e-5; L=deg2rad(40); wn=[wie*cos(L);0;-wie*sin(L)]; v=[100;0;0];
cor=2*cross(wn,v); fprintf('Coriolis east=%.4f mm/s^2\n',cor(2)*1000); % -9.3746
